package.json

```diff
@@ -20,6 +20,8 @@
   },
   "false": {},
   "dependencies": {
+    "datatables.net": "^1.10.15",
+    "datatables.net-bs": "^1.10.15",
```

Comment:
Need to remove this line, it includes Bootstrap 3. We are now pulling in Bootstrap 4 via CDN until it is released via NPM.